### PR TITLE
C1309 Edit a note on an Agreement record (ERM)

### PR DESCRIPTION
**Pull request by manowarqqq**
https://foliotest.testrail.io/index.php?/cases/view/1309

https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/4281/allure/#suites/f87695991bece2ffba5ee49eb0dabda6

### CODE REVIEW

File: cypress/support/fragments/settings/notes/settingNotes.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import uuid from 'uuid';
```
Comment (dnovikau) [inline cypress/support/fragments/settings/notes/settingNotes.js:1]: Seem that noteTypes is a better name for the module

File: cypress/e2e/agreements/agreement-notes/create-agreement-notes.cy.js [hunk 2]
```diff
@@ -1,51 +1,57 @@
...
-    Notes.deleteNoteTypeViaApi(noteTypeId);
+    SettingNotes.deleteNoteTypeViaApi(noteTypeId);
   });
 
-  it('C1308 Create a note for an Agreement record (erm)', { tags: [testTypes.extendedPath, devTeams.erm] }, () => {
-    AgreementsDetails.agreementListClick(Agreements.defaultAgreement.name);
-    AgreementsDetails.openNotesSection();
-    AgreementsDetails.verifyNotesIsEmpty();
+  it(
+    'C1308 Create a note for an Agreement record (erm)',
```
[the remark this answers is not in the document]
Comment (manowarqqq) [reply cypress/e2e/agreements/agreement-notes/create-agreement-notes.cy.js:33]: it's different file

File: cypress/support/fragments/notes/notes.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import uuid from 'uuid';
+import { REQUEST_METHOD } from '../../constants';
+
+export default {
+  createViaApi: (title, typeId, agreementId) => {
```
[the remark this answers is not in the document]
Comment (manowarqqq) [reply cypress/support/fragments/notes/notes.js:5]: done

File: cypress/support/fragments/agreements/agreementsDetails.js [hunk 4]
```diff
@@ -202,19 +210,31 @@ export default {
...
   },
 
   verifySpecialNotesRow(title, details, type) {
-    cy.expect([
-      notesList.exists(),
-    ]);
-    cy.expect(notesList.find(MultiColumnListCell({ column: 'Title and details', content: including(title) })).exists());
-    cy.expect(notesList.find(MultiColumnListCell({ column: 'Title and details', content: including(details) })).exists());
-    cy.expect(notesList.find(MultiColumnListCell({ column: 'Type', content: including(type) })).exists());
+    cy.expect([notesList.exists()]);
```
[the remark this answers is not in the document]
Comment (manowarqqq) [reply cypress/support/fragments/agreements/agreementsDetails.js:225]: done

File: cypress/support/fragments/notes/notes.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import uuid from 'uuid';
```
[the remark this answers is not in the document]
Comment (manowarqqq) [reply cypress/support/fragments/notes/notes.js:1]: there are two page notes. One is agreements > specific agreement> notes, second Settings > Notes

File: cypress/support/fragments/settings/notes/settingNotes.js (hunk 1 above)
Comment (manowarqqq) [reply cypress/support/fragments/settings/notes/settingNotes.js]: done

**Review by dnovikau — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**